features/filesystem/littlefs/littlefs/lfs_util.c

```diff
@@ -6,11 +6,10 @@
  */
 #include "lfs_util.h"
 
+
 // Only compile if user does not provide custom config
 #ifndef LFS_CONFIG
-
-
-// Software CRC implementation with small lookup table
+#ifndef __MBED__
```

Comment:
shouldn't we rather provide LFS_CONFIG than changing directly littlefs (another update might replace this?)

Reply:
@0xc0170 - Yes we can but I assumed LFS_CONFIG is apart from mbed-os changes. https://github.com/ARMmbed/mbed-os/blob/6bb2f500cddcfb707920ff866b571074cc8d0071/features/filesystem/littlefs/littlefs/lfs_util.h#L47

Reply:
LFS_CONFIG was added after we first integrated LittleFS into Mbed OS, so that's why lfs_util.c/h have been modified directly.

git subtrees handle this surprisingly gracefully, so moving to LFS_CONFIG has been low priority.

Reply:
As part of this PR, shall we update "lfs_util.c/h" or add separate files?

Reply:
> Up to you, adding separate files can be a separate PR

Will stay with this for now. Thanks